{
    "name": "libidn2",
    "ecosystem": "pypi",
    "campaign": "2025-10-bugdotexe",
    "affected_versions": [
        "2.3.2"
    ],
    "campaign_assigned_at": "2025-10-29T22:07:25.911413+00:00",
    "campaign_data_modified_at": "2025-10-29T22:07:25.911442+00:00"
}